hilti/runtime/include/fiber.h

```diff
@@ -18,12 +17,18 @@
 #include <hilti/rt/lambda.h>
 #include <hilti/rt/util.h>
 
+struct Fiber;
+
 namespace hilti::rt {
 
 namespace detail {
+extern "C" {
 void _Trampoline(void* argsp);
+}
 
 class Fiber;
+
+using FiberImpl = std::unique_ptr<::Fiber>;
```

Comment:
I'd prefer to skip this typedef as it hides that `Fiber*` and `FiberImpl` are the same thing, just with different ownership semantics.